LocalFeeder/FeederSimulator.py

```diff
@@ -748,30 +752,31 @@ def set_pv_output(self, pv_system, p, q):
         ]
         self.change_obj(command)
 
-    def get_pv_output(self, pv_system):
-        dss.PVsystems.First()
-        while True:
-            if dss.PVsystems.Name() == pv_system:
-                kw = dss.PVsystems.kW()
-                kvar = dss.PVsystems.kvar()
-            if not dss.PVsystems.Next() > 0:
-                break
-        return kw, kvar
-
     def get_max_pv_available(self, pv_system):
         dss.PVsystems.First()
         irradiance = None
         pmpp = None
         while True:
             if dss.PVsystems.Name() == pv_system:
-                irradiance = dss.PVsystems.Irradiance()
+                irradiance = dss.PVsystems.IrradianceNow()
                 pmpp = dss.PVsystems.Pmpp()
             if not dss.PVsystems.Next() > 0:
                 break
         if irradiance is None or pmpp is None:
             raise ValueError(f"Irradiance or PMPP not found for {pv_system}")
         return irradiance * pmpp
 
+    def get_available_pv(self):
+        pv_names = []
+        powers = []
+        dss.PVsystems.First()
+        while True:
+            pv_names.append(f"PVSystem.{dss.PVsystems.Name()}")
+            powers.append(dss.PVsystems.Pmpp() * dss.PVsystems.IrradianceNow())
+            if not dss.PVsystems.Next() > 0:
+                break
+        return xr.DataArray(powers, coords={"ids": pv_names})
+
```

Comment:
flag = dss.PVsystems.First()
while flag:
    flag = dss.PVsystems.Next()

implements the same while logic and is easier to digest

Reply:
Fixed.